app/helpers/spree/admin/navigation_helper_decorator.rb

```diff
@@ -18,6 +18,7 @@ def klass_for_with_sym_fallback(name)
         klass = Spree::Order if klass == :bulk_order_management
         klass = EnterpriseGroup if klass == :group
         klass = VariantOverride if klass == :Inventory
+        klass = ProductImport::ProductImporter if klass == :import
```

Comment:
can you explain how is this used? I found out this helper is used to create tab links? but I only see a single use in our codebase

Reply:
It's overriding a Spree helper, used in 5 files under `app/views/admin/shared`, wherever you see `= tab`. It probably needs a big tidy up at some point, but I didn't think it was a priority right now.

Reply:
It applies `cancan` permissions to see whether or not it should display items in the submenus for the current user.

Reply:
Would you mind adding the necessary Rubocop's `disable` to get it green in Code Climate while we are aware of the issue in the future?